book/embeds.md

```diff
@@ -954,7 +954,14 @@ class Frame:
         self.frame_height = 0
 ```
 
-And we can set those when the parent frame is laid out:
+And we can set those when the parent frame is laid out:[^no-set-needs]
+
+[^no-set-needs]: You might be surprised that I'm not calling
```

Comment:
I just remembered that chapter 14 introduces `setAttribute`. So this is not true?

Reply:
I just put up a PR to call set_needs_render from setAttribute. So just rephrase this footnote
to refer to the call there?